docs/contribution.html

```diff
@@ -93,25 +117,35 @@ <h2 id="methodes">Methodes</h2>
                     <li>private</li>
                 </ul>
             </li>
-            <li>Every methode should have a SINCE annotation (not needed if the class/struct/union already has one)</li>
+            <li>Every method should have a SINCE annotation (not needed if the class/struct/union already has one)</li>
+            <li>Exceptions are not to be used in any case</li>
+            <li>Do not force use inline methods, leave them up to the compiler</li>
+            <li>Use a pPrefix for input parameters</li>
+            <li>Do not override methods</li>
         </ul>
         <h2 id="modules">Modules</h2>
         <ul>
             <li>Modules are namespaces</li>
             <li>
-                Each moduel is a submodule of <em><strong class="mark">tge</strong></em>
+                Each module is a submodule of <em><strong class="mark">tge</strong></em>
                 <pre class="prettyprint">namespace tge {
-    namespace nio {</pre><p><em>Note that we are not using C++17 standarts for compiling so nested namespace defenitions are not allowed</em></p>
+    namespace nio {</pre>
+                <p><em>Note that we are not using C++17 standards for compiling, so nested namespace definitions are not
```

Comment:
This one is also wrong, the C++ 20 standards are approaching so we can switch that up to the new C++ 17 definition therefore see the according new modules, like Pipe.cpp or Texture.cpp